packages/@uppy/utils/package.json

```diff
@@ -4,6 +4,7 @@
   "version": "6.1.5",
   "license": "MIT",
   "type": "module",
+  "main": "./lib/index.js",
```

Comment:
adding `main` fixed the angular build. which probably means `ng-packagr` uses an older way of module resolution and was looking for the main field while building. 🤔